lib/jxl/decode_test.cc

```diff
@@ -203,8 +203,8 @@ PaddedBytes CreateTestJXLCodestream(
   io.metadata.m.color_encoding = color_encoding;
   EXPECT_TRUE(ConvertFromExternal(
       pixels, xsize, ysize, color_encoding, /*has_alpha=*/include_alpha,
-      /*alpha_is_premultiplied=*/false, bitdepth, JXL_BIG_ENDIAN,
-      /*flipped_y=*/false, &pool, &io.Main(), /*float_in=*/false));
+      /*alpha_is_premultiplied=*/false, 16, JXL_BIG_ENDIAN,
+      /*flipped_y=*/false, &pool, &io.Main(), /*float_in=*/false, 0));
```

Comment:
Can you add /*comments=*/ about numerical arguments?

Reply:
done